### PR TITLE
update exporting labels and annotations

### PR DESCRIPTION
**Pull request by mkosiarc**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- N/A JSON/YAML configuration changes are updated in the relevant schema
- N/A Changes to metadata also update the documentation for the metadata
- N/A Pull request has a link to an osbs-docs PR for user documentation updates
- N/A New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by mkosiarc** [comment]
This depends on the code from https://github.com/containerbuildsystem/osbs-client/pull/1051

**Review by tkdchen — COMMENTED** [review]
LGTM

File: atomic_reactor/tasks/sources.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ class SourceBuildTask(plugin_based.PluginBasedTask):
         ],
         exit=[
             {"name": "verify_media", "required": False},
-            {"name": "koji_import_source_container"},
+            # {"name": "koji_import_source_container"},
```
Comment (rcerven) [inline atomic_reactor/tasks/sources.py:47]: revert this testing change

File: requirements.in [hunk 2]
```diff
@@ -7,7 +7,7 @@ flatpak-module-tools >= 0.11;python_version>="3.9"
 jsonschema
 PyYAML
 ruamel.yaml
-git+https://github.com/containerbuildsystem/osbs-client@be934592d8c2a90c186bdab619a1dffb71cf62c1#egg=osbs-client
+git+https://github.com/rcerven/osbs-client@f458cb5059dc6ab0e53189217ed1cbd94066b245#egg=osbs-client
```
Comment (chmeliik) [inline requirements.in:10]: Even for testing, you will need to update this in requirements.txt as well (ideally by running `make pip-compile`)

Comment (chmeliik) [reply requirements.in]: Cachito requires the exact commit, and I think `pip-compile` doesn't support resolving branches to commits so it already has to be a commit in requirements.in

File: requirements-devel.txt [hunk 3]
```diff
@@ -54,7 +54,7 @@ koji==1.26.1
     # via -r requirements.in
 mccabe==0.6.1
     # via flake8
-osbs-client @ git+https://github.com/containerbuildsystem/osbs-client@be934592d8c2a90c186bdab619a1dffb71cf62c1
+osbs-client @ git+https://github.com/containerbuildsystem/osbs-client@3572835efc3e2a7fbde414391d2bf6bf1108e915
```
Comment (lkolacek) [inline requirements-devel.txt:57]: There seems to be already newer version 1b5823ec62be281cea8e096a034997ff86cdab86

File: atomic_reactor/config.py [hunk 4]
```diff
@@ -91,7 +91,7 @@ def get_openshift_session(config, namespace):
         'namespace': namespace,
         'use_auth': False,
         'conf_file': None,
-        'openshift_url': config.openshift['url'],
+        'openshift_url': KUBERNETES_API_URL,
```
[the remark this answers is not in the document]
Comment (rcerven) [reply atomic_reactor/config.py:94]: I wouldn't hardcode it even for that case, and just use url from reactor-config-map

**Review by lkolacek — APPROVED** [review]

**Review by rcerven — APPROVED** [review]

**Review by ben-alkov — APPROVED** [review]

**Pull request merged**